packages/dioxus-blitz/src/window.rs

```diff
@@ -152,10 +167,78 @@ impl<'a, Doc: DocumentLike> View<'a, Doc> {
                 // modifiers,
                 ..
             } => {
-                let tao::dpi::LogicalPosition::<f32> { x, y } = position.to_logical(2.0);
-                if self.renderer.mouse_move(x, y) {
-                    self.request_redraw();
+                let changed = if let RenderState::Active(state) = &self.renderer.render_state {
+                    // todo: this is a hardcode of the hidpi scale, we should get this from the window
+                    let tao::dpi::LogicalPosition::<f32> { x, y } = position.to_logical(state.window.scale_factor());
+
+                    dbg!(self.renderer.mouse_move(x, y))
+                } else {
+                    false
+                };
+
+                if changed {
+                    let cursor = self.renderer.get_cursor();
+                    dbg!(cursor);
+
+                    if let Some(cursor) = cursor {
+                        use style::values::computed::ui::CursorKind;
+                        use tao::window::CursorIcon as TaoCursor;
+                        let tao_cursor = match cursor {
+                            CursorKind::None => todo!("set the cursor to none"),
+                            CursorKind::Default => TaoCursor::Default,
```

Comment:
Maybe a `_ => cursor` catch-all for the cases where no mapping is happening. Normally I like to write out all the variants, but it doesn't seem important here...